packages/matchbox/src/components/Slider/styles.js

```diff
@@ -0,0 +1,131 @@
+import { tokens } from '@sparkpost/design-tokens';
+import { StyledSlider } from './Slider';
+
+export const slider = props => {
+  let cursor = 'pointer';
+
+  if (props.disabled) {
+    cursor = 'not-allowed';
+  }
+
+  return `
+    padding: ${tokens.spacing_400} 0;
```

Comment:
I think here to get 40px you'd have to do `1.25rem 0`, not a token though, which is fine for now